labgrid/resource/onewireport.py

```diff
@@ -0,0 +1,11 @@
+import attr
+
+from ..factory import target_factory
+from .common import Resource
+
+
+@target_factory.reg_resource
+@attr.s
+class OneWirePort(Resource):
+    portname = attr.ib(validator=attr.validators.instance_of(str))
+    url = attr.ib(validator=attr.validators.instance_of(str))
```

Comment:
This should be called OneWirePIO and have host and path properties. Also add a docstring giving an example path.